test/unit/dataSourceProvider.js

```diff
@@ -224,6 +224,7 @@ describe('Provide in Sources', function () {
         tile.parent = { pendingSubdivision: false };
         featureLayer.source.uid = 8;
         featureLayer.mergeFeatures = true;
+        featureLayer.cache.data.clear();
```

Comment:
Curious: why is it needed ?

Reply:
Because is the same layer used in all the tests, we need to clear cache to avoid return the same value.